setup.py

```diff
@@ -41,6 +41,11 @@ def get_requirements():
     ],
     install_requires=get_requirements(),
     python_requires=">=2.6",
+    entry_points={
```

Comment:
This should not be needed, this entry point is meant to be declared in pubtools-exodus.